### ISSUE TITLE
Version .NET Core Assemblies fails with "Unexpected token =" on TFS 2015.4

### ISSUE THREAD

**Issue by HolisticDeveloper** [rfennell/AzurePipelines#457]
### Azure DevOps Extension you are using
Manifest Versioning Build Tasks -> Version .NET Core Assemblies task

### Where are you running it?
Azure DevOps Server (TFS 2015.4)

### Version of Extension/Task
2.8.3

### Expected behaviour and actual behaviour
Expected behavior:
Updates csproj files.

Actual behavior:
Errors in the log:

```
2019-03-18T02:16:14.8020741Z Set workingFolder to default: C:\agent\tasks\VersionDotNetCoreAssemblies\2.8.3
2019-03-18T02:16:16.7246057Z C:\agent\tasks\VersionDotNetCoreAssemblies\2.8.3\AppyVersionToAssembliesFunctions.js:113
2019-03-18T02:16:16.7255821Z function ProcessFile(file, field, newVersion, addDefault = false) {
2019-03-18T02:16:16.7265585Z                                                          ^
2019-03-18T02:16:16.7285113Z SyntaxError: Unexpected token =
2019-03-18T02:16:16.7285113Z     at exports.runInThisContext (vm.js:53:16)
2019-03-18T02:16:16.7294877Z     at Module._compile (module.js:387:25)
2019-03-18T02:16:16.7304641Z     at Object.Module._extensions..js (module.js:422:10)
2019-03-18T02:16:16.7314405Z     at Module.load (module.js:357:32)
2019-03-18T02:16:16.7324169Z     at Function.Module._load (module.js:314:12)
2019-03-18T02:16:16.7333933Z     at Module.require (module.js:367:17)
2019-03-18T02:16:16.7343697Z     at require (internal/module.js:16:19)
2019-03-18T02:16:16.7343697Z     at Object.<anonymous> (C:\agent\tasks\VersionDotNetCoreAssemblies\2.8.3\ApplyVersionToAssemblies.js:3:44)
2019-03-18T02:16:16.7363225Z     at Module._compile (module.js:413:34)
2019-03-18T02:16:16.7372989Z     at Object.Module._extensions..js (module.js:422:10)
```

### Steps to reproduce the problem
1. Use TFS 2015.4 with a local agent.
2. Create and configure a new build definition.
3. Add "Version .NET Core Assemblies" task with default values.
4. Save and queue the build.

**Comment by HolisticDeveloper** [comment]
It appears that this issue is caused by the TFS 2015 agent using Node 5, which only supports ES5. Default parameter values are only supported in Node 6 or later. Changing the TypeScript compiler configuration to output ES5 resolves the issue. I'll be sending a PR with a fix.

**Comment by rfennell** [comment]
Released as 2.9.6
